### DISCUSSION TITLE
openwrt无法获取ipv6

### DISCUSSION THREAD

**Discussion by mjc88** [coolsnowwolf/lede#11212] [Bugs / 问题反馈]
求助各位大佬，使用lede源码进行编译后，拨号后可以正常上网。但是无法获取ipv6，这是啥原因啊？
之前使用他人编译好的是可以正常获取ipv6的，在WAN与LAN的配置和老版本的是一模一样的。
: - (

**Comment by licsber** [comment]
lede默认不开ipv6的支持 自行加config

```bash
 echo 'CONFIG_PACKAGE_ipv6helper=y' >> .config
          echo 'CONFIG_PACKAGE_ip6tables-extra=y' >> .config
          echo 'CONFIG_PACKAGE_ip6tables-mod-nat=y' >> .config
```

**Reply by djnmk12** [reply]
你好分别运行这三行代码就可以了吗？我运行了不行啊

**Reply by licsber** [reply]
@djnmk12 用github actions编译的话参考我fork的仓库 不然就是你本地代码的根目录 直接shell运行就行

**Comment by King-Of-Knights** [comment]
看看extra package 里面 ipv6helper 有没有勾上
